### PR TITLE
Add support for imx8mn-lpddr4-evk/sec

### PR DESCRIPTION
**Pull request by ricardosalveti**
Quite similar to imx8mn-ddr4-evk (most options were duplicated), but since we don't have an override that is evk specific we have to use board specific settings instead.

This is not yet tested, having this PR open for when we get the hardware in hands.

### CODE REVIEW

File: meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/imx8mn-lpddr4-evk-sec/lmp.cfg [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+CONFIG_FIOVB=y
+CONFIG_CMD_FIOVB=y
+CONFIG_CMD_XIMG=y
+CONFIG_CMD_SECONDARY_BOOT=y
+CONFIG_SUPPORT_EMMC_RPMB=y
+CONFIG_SUPPORT_EMMC_BOOT=y
+# CONFIG_ENV_IS_IN_MMC is not set
+# CONFIG_ENV_IS_IN_FAT is not set
+CONFIG_ENV_IS_NOWHERE=y
+CONFIG_ENV_FAT_INTERFACE="mmc"
```
Comment (mike-scott) [inline meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/imx8mn-lpddr4-evk-sec/lmp.cfg:55]: I think these `CONFIG_ENV_FAT_*` settings can be removed from the `-sec`

**Review by mike-scott — COMMENTED** [review]
Minor nit for -sec mfgtool `lmp.cfg`.  Otherwise this looks good (haven't tested).

**Review by MrCry0 — APPROVED** [review]
LGTM (with mike's note, not tested)

**Review by angolini — APPROVED** [review]

File: meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/imx8mn-lpddr4-evk-sec/lmp.cfg (hunk 1 above)
Comment (ricardosalveti) [reply meta-lmp-bsp/recipes-bsp/u-boot/u-boot-fio/imx8mn-lpddr4-evk-sec/lmp.cfg]: Sure, done.

**Comment by ricardosalveti** [comment]
Will merge and enable on ci-scripts so we can get official builds for it, and then work with @mwasilew to get it tested correctly.

**Pull request merged**